test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/testproviders/ProcessTestProvider.java

```diff
@@ -50,12 +50,14 @@ public static void testProcess(KieServerDeployment kieServerDeployment) {
         ProcessServicesClient processClient = KieServerClientProvider.getProcessClient(kieServerDeployment);
         UserTaskServicesClient taskClient = KieServerClientProvider.getTaskClient(kieServerDeployment);
 
+        int tasksCount = taskClient.findTasks(Constants.User.YODA, 0, 10).size();
+
         try {
             Long userTaskPid = processClient.startProcess(containerId, Constants.ProcessId.USERTASK);
             assertThat(userTaskPid).isNotNull();
 
             List<TaskSummary> tasks = taskClient.findTasks(Constants.User.YODA, 0, 10);
-            assertThat(tasks).isNotNull().hasSize(1);
+            assertThat(tasks).isNotNull().hasSize(tasksCount + 1);
```

Comment:
Please raise page size in case you do size comparison, ideally to some size which couldn't be reachable (like 10000).